espnet/nets/pytorch_backend/e2e_asr_transformer.py

```diff
@@ -613,6 +637,67 @@ def recognize(self, x, recog_args, char_list=None, rnnlm=None, use_jit=False):
         )
         return nbest_hyps
 
+    def recognize_mask_ctc(self, x, recog_args, char_list=None):
```

Comment:
```suggestion
    def recognize_nat(self, x, recog_args, char_list=None):
```
And you can call `recognize_mask_ctc` inside `recognize_nat`.